src/test/java/org/springframework/cloud/deployer/spi/kubernetes/RunAbstractKubernetesDeployerTests.java

```diff
@@ -152,65 +121,4 @@ public void requestMemory_deploymentProperty_usesDeploymentProperty() throws Exc
 		Map<String, Quantity> requests = kubernetesDeployer.deduceResourceRequests(deploymentRequest);
 		MatcherAssert.assertThat(requests.get("memory"), is(new Quantity("256Mi")));
 	}
-
-	@Test
-	public void limitMemory_commonDeploymentProperty_noUnit() throws Exception {
-		deploymentProperties.put(AppDeployer.MEMORY_PROPERTY_KEY, "512");
-		Map<String, Quantity> limits = kubernetesDeployer.deduceResourceLimits(deploymentRequest);
-		MatcherAssert.assertThat(limits.get("memory"), is(new Quantity("512Mi")));
-	}
-
-	@Test
-	public void limitMemory_commonDeploymentProperty_mSuffix() throws Exception {
-		deploymentProperties.put(AppDeployer.MEMORY_PROPERTY_KEY, "512m");
-		Map<String, Quantity> limits = kubernetesDeployer.deduceResourceLimits(deploymentRequest);
-		MatcherAssert.assertThat(limits.get("memory"), is(new Quantity("512Mi")));
-	}
-
-	@Test
-	public void limitMemory_commonDeploymentProperty_gSuffix() throws Exception {
```

Comment:
Just curious why we removed gsuffix, wholeNumber and fraction tests

Reply:
the tests removed were ones using deprecated methods / properties. there are similar assertions that remained in the test class but use the non-deprecated equivalent methods / properties